src/index.js

```diff
@@ -1,20 +1,53 @@
+//npm packages
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import StockView from './StockView';
+import { ThemeProvider } from '@material-ui/core'
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Cookies from 'universal-cookie'
 import reportWebVitals from './reportWebVitals';
+//Components
+import theme from './components/theme'
+import Header from './components/Header'
+import App from './App';
+import StockView from './StockView';
+//Styling
+import './index.css';
+
+const cookies = new Cookies()
+if(!cookies.get('theme'))
+  cookies.set('theme', 'light', {path: '/'})
+
+let currentTheme = cookies.get('theme');
+
+const themeSwitcher = (newTheme) => {
+  currentTheme = newTheme.target.checked ? 'dark' : 'light';
+  cookies.set('theme', currentTheme, {path: '/'});
+
+  ReactDOM.render(
+    <React.StrictMode>
+    <Content />
+    </React.StrictMode>,
+    document.getElementById('root')
+  );
+}
+
+function Content() {
+  return (
+      <ThemeProvider theme={theme(currentTheme)}>
+        <Header switcher={themeSwitcher} defaultSwitch={currentTheme == 'dark' ? true : false} />
```

Comment:
Line 37:70:  Expected '===' and instead saw '=='  eqeqeq
doesn't deploy to server xdd